### ISSUE TITLE
Launch configuration dialogs at the end of installation

### ISSUE THREAD

**Issue by GoogleCodeExporter** [3s3s/lavfilters#168]
```
I would like to see an option to have checkmarks at the end of the installation 
process, marking which of the three different configuration dialogs should be 
launched once the installer finishes.
Basically, the same thing as in FFDShow and numerous other installers.

Obviously low priority, but shouldn't be too hard I guess (sound like a common 
installer script modification).
```

Original issue reported on code.google.com by `adam...@gmail.com` on 12 Jan 2012 at 12:22
- Blocked on: #176
